lib/index.js

```diff
@@ -1559,21 +1567,22 @@ module.exports = (function() {
         }
         else{
             if(typeof(imageObj) === 'string'  || imageObj instanceof String){
+                imageObj = imageObj.toString();
                 //if(this.isUrl(imageObj)){
                     // TODO
                 //}else{
                     if("imageRootPath" in this.option && fs.existsSync(this.option.imageRootPath + "/" + imageObj)){
                         //get the Absolute path file
-                        return new Buffer.from(fs.readFileSync(this.option.imageRootPath + "/" + imageObj, { encoding: 'base64' }), 'base64');
+                        return Buffer.from(fs.readFileSync(this.option.imageRootPath + "/" + imageObj, { encoding: 'base64' }), 'base64');
```

Comment:
@jdugh Also this one is annoying error. Maybe you should be aware that `new` was not needed.